package.json

```diff
@@ -9,10 +9,11 @@
     "test": "jest",
     "lint": "eslint src --ext ts",
     "typescript": "tsc",
-    "prettier": "prettier src/**/* --write",
+    "prettier": "prettier 'src/**/*' --write",
```

Comment:
without the single quote, bash will expand this glob instead of passing it into the prettier CLI. We need the prettier CLI to do the glob expansion in a reliable way.